### PR TITLE
chore: move out renderer typings from styles package

### PR DESCRIPTION
**Pull request by layershifter**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

A follow up from #13602. As we agreed to move out typings related to renderers this PR does this, typings were moved to `@fluentui/react-northstar-styles-renderer`.

### CODE REVIEW

File: packages/fluentui/react-northstar-fela-renderer/src/createFelaRenderer.tsx [hunk 1]
```diff
@@ -77,8 +78,6 @@ const rendererConfig = {
   ],
 };
 
-type FelaRendererParam = Omit<RendererParam, 'RendererParam'> & { theme: { direction: 'rtl' | 'ltr' } };
```
Comment (layershifter) [inline packages/fluentui/react-northstar-fela-renderer/src/createFelaRenderer.tsx:80]: Was moved to `./types`.

File: packages/fluentui/react-northstar-fela-renderer/src/felaDisableAnimationsPlugin.ts [hunk 2]
```diff
@@ -24,8 +24,8 @@ const animationProps: (keyof ICSSInJSStyle)[] = [
 const felaDisableAnimationsPlugin = (
   styles: ICSSInJSStyle,
   type: string,
-  renderer?: Renderer,
-  props?: Partial<RendererParam>,
+  renderer?: FelaRenderer,
+  props?: Partial<FelaRendererParam>,
```
Comment (layershifter) [inline packages/fluentui/react-northstar-fela-renderer/src/felaDisableAnimationsPlugin.ts:28]: Oops, previously I used wrong types.

File: packages/fluentui/react-northstar-fela-renderer/src/types.ts [hunk 3]
```diff
@@ -1,14 +1,15 @@
 import { IRenderer } from 'fela';
+import { RendererParam } from '@fluentui/react-northstar-styles-renderer';
 import { ICSSInJSStyle } from '@fluentui/styles';
 
-export type Renderer = IRenderer & {
-  cache: Record<string, RendererChange>;
+export type FelaRenderer = IRenderer & {
```
Comment (layershifter) [inline packages/fluentui/react-northstar-fela-renderer/src/types.ts:5]: I added `Fela` prefix to these types to clearly identify them

File: packages/fluentui/react-northstar-fela-renderer/tsconfig.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
   "include": ["src", "test"],
   "references": [
     {
-      "path": "../styles"
+      "path": "../react-northstar-styles-renderer"
```
Comment (mnajdova) [inline packages/fluentui/react-northstar-fela-renderer/tsconfig.json:15]: This was the dependency graph I was expecting 👍

File: packages/fluentui/react-northstar-fela-renderer/package.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
   "bugs": "https://github.com/microsoft/fluentui/issues",
   "dependencies": {
     "@babel/runtime": "^7.7.6",
-    "@fluentui/styles": "^0.49.0",
+    "@fluentui/react-northstar-styles-renderer": "^0.49.0",
```
Comment (mnajdova) [inline packages/fluentui/react-northstar-fela-renderer/package.json:8]: This was the dependency graph I was expecting 👍

**Pull request merged**